nextflow-modules/modules/skesa/main.nf

```diff
@@ -10,7 +10,7 @@ process skesa {
     path "*versions.yml"               , emit: versions
 
   when:
-    task.ext.when && platform == "illumina"
```

Comment:
Yeah it doesn't work otherwise. Takes the platform from the input csv and there's a check to see that the platform is one of the following: ["illumina", "nanopore", "pacbio", "iontorrent"] or it spits out an error message.